website/docs/language/state/locking.mdx

```diff
@@ -14,7 +14,7 @@ others from acquiring the lock and potentially corrupting your state.
 State locking happens automatically on all operations that could write
 state. You won't see any message that it is happening. If state locking fails,
 Terraform will not continue. You can disable state locking for most commands
-with the `-lock` flag but it is not recommended.
+with the `-lock=false` flag but it is not recommended.
```

Comment:
```suggestion
with the `-lock=false` flag, but we do not recommend it.
```

Suggest changing "it is not recommended" to "we do not recommend it" to avoid a passive voice style issue.